services/notification/types/runwebhookdelivery.go

```diff
@@ -15,6 +15,14 @@ const (
 	DeliveryStatusDeliveryError DeliveryStatus = "deliveryError"
 )
 
+func DeliveryStatusFromStringSlice(slice []string) []DeliveryStatus {
+	dss := make([]DeliveryStatus, len(slice))
```

Comment:
What happens when the query contains:
* a wrong delivery status?
* an empty delivery status?

Reply:
at the moment if the query contains a wrong or empty status the function will return it inside of the DeliveryStatus slice, the db query will ignore these wrong values.. this is the same as RunsHandler.

I can check the values in DeliveryStatusFromStringSlice and add an error return.